app/javascript/components/search/controls/FacetsPanel.js

```diff
@@ -18,15 +18,22 @@ export default function FacetsPanel() {
   const defaultFacets = searchFacetContext.facets.filter(facet => defaultFacetIds.includes(facet.id))
   const moreFacets = searchFacetContext.facets.filter(facet => moreFacetIds.includes(facet.id))
   return (
-    <>
-      <CombinedFacetControl controlDisplayName="cell type" facetIds={['cell_type', 'cell_type__custom']}/>
+    <div>
+      <span
+        className='badge metadata-search search-title'
+        title='Search through metadata for values that may not appear in titles/descriptions, like cell type or disease'
+        data-toggle='tooltip'
+      >
+        Data search
+      </span>
```

Comment:
we should move the (?) advanced search popup here

Reply:
Can do.  If we do this, I think we should remove the tooltip info and put it into the modal.  There's no reason to have both.